index.html

```diff
@@ -51,28 +51,28 @@ <h2 class="sub-txt">LET'S CONNECT</h2>
             <div class="works1" id="portfolio">
                 <img src="image/works1-big.png" alt="Avocode Snapshoot">                    
                 <div class="work-desc">
-                    <h2>Tonic</h2>
+                    <h2 class="title-works">Tonic</h2>
                     <div class="sub-box">
                         <h3>CANOPY</h3>
                         <ul>
```

Comment:
- [ ] One of the requirements in this milestone is to create all the project cards dynamically from the javascript array that you have created in the `popup.js` file. so that file has two purposes. the first purpose is to generate project cards dynamically when the page loads and the second purpose is to build popup modals. so kindly consider removing all the HTML you have created inside `<section class="works" name="works">` tag. then you can get this tag in javascript and populate the cards with dynamic data.
   -  You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.

Reply:
@Zuhaib042, The cards were inserted dynamically
![image](https://user-images.githubusercontent.com/110146573/225828295-f07d6190-9acc-4b6d-9fac-2bd36bdab787.png)